controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -112,7 +138,7 @@ bool master_thread::init()
         }
     }
 
-    return socket_thread::init();
+    return true;
```

Comment:
This change should have been part of "controller: Make son_master_thread to be derived from btl" commit.

Reply:
what? the returning of `true`? it is not critical...

Reply:
Not critical.